### PR TITLE
feat: added dashboard checkbox functionality

### PR DESCRIPTION
**Pull request by willfpearce**
From #32:  I got it working with made up id's for each row, which I think is a good solution because I'll I needed to do was add an id property to the data objects, which kind of simulates getting the id from the database. Also, I added a new js file "dashboard.js" because I assume we'll need more interactivity for the page later

### CODE REVIEW

**Comment by willfpearce** [comment]
in order to get passed the "no-used-vars" check in the linter I had to call the checkboxClickHandler function within the js file, which kind of stinks

File: public/js/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const blogTableSelections = []
```
Comment (zeim839) [inline public/js/dashboard.js:1]: change this to
```js
var blogTableSelections = [] // eslint-disable-line
```
This should disable the linter.

File: public/js/dashboard.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  const idIndex = blogTableSelections.indexOf(id)
+  if (idIndex !== -1) {
+    blogTableSelections.splice(idIndex, 1)
+  } else {
+    blogTableSelections.push(id)
+  }
+}
+
+checkboxClickHandler('1')
+checkboxClickHandler('1')
```
Comment (zeim839) [inline public/js/dashboard.js:13]: Do we need this?

**Review by zeim839 — CHANGES_REQUESTED** [review]
Could you also implement the checkbox in the header? You can do this by iterating the table's children and adding each row to the array by its element ID.
See [example](https://www.w3schools.com/jsref/prop_element_children.asp)

File: public/js/dashboard.js (hunk 1 above)
Comment (willfpearce) [reply public/js/dashboard.js:1]: 👍

File: public/js/dashboard.js (hunk 2 above)
Comment (willfpearce) [reply public/js/dashboard.js:13]: No that was the fix for the linter.

File: public/js/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const blogTableSelections = [] // eslint-disable-line
```
[the remark this answers is not in the document]
Comment (willfpearce) [reply public/js/dashboard.js]: 👍

**Pull request merged**